gossip/state/state.go

```diff
@@ -761,6 +761,10 @@ func (s *GossipStateProviderImpl) addPayload(payload *proto.Payload, blockingMod
 	}
 
 	if !blockingMode && payload.SeqNum-height >= uint64(s.config.StateBlockBufferSize) {
+		if s.straggler(height, payload) {
+			s.logger.Warningf("Detected a distant block sequence, we are lagging behind and state transfer is disabled in configuration " +
```

Comment:
It is possible to log the channel as well as how far behind the peer actually is?